[2025-03-15T13:10:20.546+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T13:10:20.563+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T13:10:18.937484+00:00 [queued]>
[2025-03-15T13:10:20.572+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T13:10:18.937484+00:00 [queued]>
[2025-03-15T13:10:20.574+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-15T13:10:20.588+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-15 13:10:18.937484+00:00
[2025-03-15T13:10:20.595+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=425) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T13:10:20.597+0000] {standard_task_runner.py:72} INFO - Started process 427 to run task
[2025-03-15T13:10:20.598+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'manual__2025-03-15T13:10:18.937484+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmpzd99milw']
[2025-03-15T13:10:20.601+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask api
[2025-03-15T13:10:20.647+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api manual__2025-03-15T13:10:18.937484+00:00 [running]> on host 157b2d7c5f40
[2025-03-15T13:10:20.872+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T13:10:18.937484+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T13:10:18.937484+00:00'
[2025-03-15T13:10:20.873+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-15T13:10:21.135+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-15T13:10:21.155+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-15T13:10:21.156+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=manual__2025-03-15T13:10:18.937484+00:00, execution_date=20250315T131018, start_date=20250315T131020, end_date=20250315T131021
[2025-03-15T13:10:21.215+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T13:10:21.231+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T13:10:21.233+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
